green/result.py

```diff
@@ -256,44 +264,39 @@ def __init__(self, start_callback=None, finalize_callback=None):
             "test_time",
         ]
         self.failfast = False  # Because unittest inspects the attribute
-        self.reinitialize()
-
-    def reinitialize(self):
-        self.shouldStop = False
         self.collectedDurations = []
         self.errors = []
         self.expectedFailures = []
         self.failures = []
         self.passing = []
         self.skipped = []
         self.unexpectedSuccesses = []
+        self.reinitialize()
```

Comment:
FYI, linters are not too happy when properties are declared outside init, moving the lists declarations to `__init__()` helps with that and the list can be made empty again with `.clear()`.